doc/samtools.1

```diff
@@ -937,18 +937,56 @@ block compression.
 CRAM output only; defaults to 0 (off).  Permits use of lzma in CRAM
 block compression.
 .TP
+.BI use_fqz= 0|1
+CRAM >= 3.1 output only; enables and disables the fqzcomp quality
```

Comment:
These could be pretty-printed as `≥`, which makes a difference to the HTML view on http://htslib.org/doc and also in `man samtools` if your terminal is in an UTF-8 locale (which it probably is, these days):

```suggestion
CRAM \(>= 3.1 output only; enables and disables the fqzcomp quality
```

Reply:
Thanks John.